media-source-respec.html

```diff
@@ -2723,7 +2722,17 @@
         </thead>
         <tbody>
           <tr>
-            <td>April 15 2016</td>
+            <td>April 18 2016</td>
```

Comment:
nit/standard-operating-procedure reminder: Rebase, update date here, publishDate above, and regenerate index.html.